packages/core/types/src/common/config-module.ts

```diff
@@ -354,11 +364,39 @@ export type ProjectConfigOptions = {
    */
   redisOptions?: RedisOptions
 
+  /**
+   * Specify the options for storing session data to dynamoDB. Make
+   * sure to install the following dependencies and set the following
+   * variables too.
+   *
+   * - @aws-sdk/client-dynamodb@^3.218.0
+   * - connect-dynamodb@^3.0.5
+   *
+   * Environment variables to set
+   * - AWS_REGION=<value>
+   * - ENDPOINT=<value>
+   * - AWS_ACCESS_KEY_ID=<value>
+   * - AWS_SECRET_ACCESS_KEY=<value>
+   */
+  dynamodbOptions?: {
```

Comment:
thought(non-blocking): Should we nest this within `sessionStore` to make it more obvious what it is intended for?

Reply:
We have similar setup for redis. So should be also nest that within the `sessionStore`? Because there could be case where I want to use separate Redis stores for events and for storing sessions

Reply:
> Because there could be case where I want to use separate Redis stores for events and for storing sessions

I am not sure I understand how this is related to the config–can you elaborate?

Reply:
Currently when we pick Redis as the session store for storing the session data, we rely on top-level `projectConfig.redisUrl` and `projectConfig.redisOptions` config variables. 

So for consistency both Redis and DynamoDB should be at the same level

Reply:
Redis being top-level is more a legacy thing than intended actually. I would be OK with moving the dynamo DB options to `sessionStore` and keep Redis as is to avoid breaking changes.<br><br>The thinking here is that suddenly having `dynamodbOptions` live top-level in the project config might be cause of confusion. What is it for and do I need it?

This is not a strong opinion though, so happy to go with whatever.

Reply:
Yup happy to move it to the `sessionStore` 👍